### PR TITLE
v1.0.0 release readme changes

### PR DESCRIPTION
**Pull request by crookse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,215 +103,212 @@ test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (4ms)
...
+Rhum.asserts.assertEquals(true, false); // fail
+```
 
-})
-Rhum.run();
+#### `Rhum.mocks`
+
+An object of functions to help you mock objects.
+
+##### `Rhum.mocks.ServerRequest`
```
Comment (ebebbington) [inline README.md:136]: Should `Rhum.mocks` be `###` and `Rhum.mocks.ServerRequest` be `####`? Looks a little odd with `Rhum.mocks` being a smaller font size than the method headings

Comment (crookse) [reply README.md:136]: Oh right

Comment (crookse) [reply README.md:136]: oh wait. no because Rhum.mocks is a subheading of Properties. Properties is ### and Rhum.mocks is ####

Comment (ebebbington) [reply README.md:136]: Ignore me, i can't see.. They are both the same size :S

**Pull request merged**